src/main/scala/nl.knaw.dans.easy.sword2/DepositProperties.scala

```diff
@@ -103,6 +103,11 @@ class DepositProperties(depositId: DepositId, depositorId: Option[String] = None
     this
   }
 
+  def setOtherIdentifier(id: String, version: String): Try[DepositProperties] = Try {
+    if (id.nonEmpty) properties.setProperty("dataverse.other-id", id)
+    if (version.nonEmpty) properties.setProperty("dataverse.other-id-version", version)
```

Comment:
```suggestion
    id.foreach(properties.setProperty("dataverse.other-id", _))
    version.foreach(properties.setProperty("dataverse.other-id-version", _))
```

Reply:
`id` and `version` are Strings.  You could do `Option(id).foreach...`  but then you would get this when the strings are empty: 
```
dataverse.other-id =
dataverse.other-id-version =
```

When we leave the code as it is now,  these properties come into the properties file only when they are given in bag-info.txt

Reply:
sorry, assumed it where options.